### PR TITLE
 added batched cluster query

### PR DESCRIPTION
**Pull request by xrajesh**

### CODE REVIEW

File: start.py [hunk 1]
```diff
@@ -15,6 +15,7 @@
 start_date = end_date - timedelta(days=days_delta - 1)
 print("Metrics from " ,start_date-timedelta(days=1), "is used to compute recommendation")
 tolerance=int(os.environ.get('TOLERANCE', '0'))
+batch_size=int(os.environ.get('BATCH_SIZE', '1'))
```
Comment (bjoydeep) [inline start.py:18]: We should add it to the recommend.sh, right @xrajesh ?

File: batch_worker.py [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+
+            cpu_usage_max.append(
+                self.process_metrics_batch(CPU_USAGE_MAX, cluster_ids_filter, epoch)
+            )
+            memory_usage_max.append(
+                self.process_metrics_batch(MEMORY_USAGE_MAX, cluster_ids_filter, epoch)
+            )
+
+        df_cpu_usage_max = self.get_grouped_aggregation(cpu_usage_max, "max")
+        df_memory_usage_max = self.get_grouped_aggregation(memory_usage_max, "max")
```
Comment (bjoydeep) [inline batch_worker.py:68]: Are these doing a max of max? The query over which it max-es already includes a max_over_time, right?

File: start.py (hunk 1 above)
Comment (bjoydeep) [reply start.py:18]: As I think more, I think what you have now is fine. Unless you have second thoughts, we can resolve this

File: batch_worker.py (hunk 2 above)
Comment (xrajesh) [reply batch_worker.py:82]: @bjoydeep This is to get the max from each of the days MAX.

**Review by bjoydeep — APPROVED** [review]
/lgtm

**Review by bjoydeep — APPROVED** [review]

**Pull request merged**